eng/common/Extract-ReleaseNotes.ps1

```diff
@@ -0,0 +1,47 @@
+# given a CHANGELOG.md file, extract the relevant info we need to decorate a release
+param (
+  [Parameter(Mandatory = $true)]
+  [String]$ChangeLogLocation
+)
+
+$RELEASE_TITLE_REGEX = "(?<releaseNoteTitle>^\#+.*(?<version>\b\d+\.\d+\.\d+([^0-9\s][^\s:]+)?))"
```

Comment:
This parsing logic is duplicated with the Update-Change-Log.ps1 script we should try and share this logic in a common ps1 file that can be imported and reused in the different change-log scripts.

Reply:
Yes, I plan to modify the Update-ChangeLog.ps1 to call this script rather than reading changelogs itself.